saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetailsMeta.jsp

```diff
@@ -0,0 +1,211 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
+
+
+<spring:message code="judgmentDetails.meta.keywords" var="detailsMetaKeywords" />
+
+<%-- JudgmentType --%>
+<c:if test="${!empty judgment.judgmentType}" >
+    <c:if test="${!empty detailsMetaKeywords}" >
```

Comment:
I think you should not check if detailsMetaKeywords var is empty. You know it's not.